Sources/ArcGISToolkit/Components/Form/FormInputs/ComboBoxInput.swift

```diff
@@ -0,0 +1,245 @@
+            guard let current = codedValues.first(where: { $0.name == element.value }),
+                  current != newValue else {
+                return
+            }
+            requiredValueMissing = element.isRequired && newValue == nil
+            featureForm?.feature.setAttributeValue(newValue?.code, forKey: element.fieldName)
+        }
+    }
+    
+    /// The root of the picker view.
+    ///
+    /// Adds navigation context to support toolbar items and other visual elements in the picker.
+    /// - Note `NavigationView` is deprecated after iOS 17.0.
+    @ViewBuilder var pickerRoot: some View {
+        if #available(iOS 16, macCatalyst 16, *) {
+            NavigationStack {
+                picker
+            }
+        } else {
+            NavigationView {
+                picker
+            }
+        }
+    }
+    
+    /// The view that allows the user to filter and select coded values by name.
+    var picker: some View {
+        VStack {
+            Text(element.description)
+                .foregroundColor(.secondary)
+                .font(.subheadline)
+                .padding(.horizontal)
+                .frame(maxWidth: .infinity, alignment: .leading)
+            Divider()
+            List {
+                if element.value.isEmpty && !element.isRequired {
+                    if input.noValueOption == .show {
+                        HStack {
+                            Button(input.noValueLabel.isEmpty ? String.noValue : input.noValueLabel) {
+                                selectedValue = nil
+                            }
+                            Spacer()
+                            if selectedValue == nil {
+                                Image(systemName: "checkmark")
+                                    .foregroundColor(.accentColor)
+                            }
+                        }
+                    }
+                }
+                ForEach(matchingValues, id: \.self) { codedValue in
+                    HStack {
+                        Button(codedValue.name) {
+                            selectedValue = codedValue
+                        }
+                        Spacer()
+                        if codedValue == selectedValue {
+                            Image(systemName: "checkmark")
+                                .foregroundColor(.accentColor)
+                        }
+                    }
+                }
+            }
+            .listStyle(.plain)
+            .searchable(text: $filterPhrase, placement: .navigationBarDrawer, prompt: .filter)
+            .navigationTitle(element.label)
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    Button {
+                        isPresented = false
+                    } label: {
+                        Text.done
+                            .bold()
```

Comment:
Why are some of the `Text.done` instances `.semiBold()` and this one is `.bold()`?